cmds/ecr-repositories/ecr-repositories.go

```diff
@@ -37,3 +43,22 @@ var queryCmd = &cobra.Command{
 		fmt.Println(repositories)
 	},
 }
+
+var createECRCmd = &cobra.Command{
+	Use:   "create [NAME]",
+	Short: "creates a new ecr repository",
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 0 {
+			fmt.Println("Missing id")
```

Comment:
Maybe name instead of id, to avoid any confusion, since we call it name in our help text